core/kernel/notif.c

```diff
@@ -307,7 +307,7 @@ TEE_Result notif_itr_register(struct notif_itr *notif)
 {
 	unsigned int itr_num = 0;
 	uint32_t exceptions = 0;
-	const struct notif_itr_ops *ops = NULL;
+	const __maybe_unused struct notif_itr_ops *ops = NULL;
```

Comment:
I believe this should go into an earlier commit.

Reply:
right, i'll need to fix that when squashing fixup commits.